boskos/common/common.go

```diff
@@ -58,6 +60,34 @@ type Item interface {
 	GetName() string
 }
 
+// Duration is a wrapper around time.Duration that parses times in either
+// 'integer number of nanoseconds' or 'duration string' formats and serializes
+// to 'duration string' format.
+type Duration struct {
```

Comment:
We already have metav1.Duration (https://github.com/kubernetes/test-infra/blob/master/prow/config/config.go#L130), maybe use that, instead of reinvent the wheel here

Reply:
I actually copied it. the only difference is it has to be type struct Duration {*time.Duration} instead of Duration {timeDuration} as I am using a *time.Duration.